File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file270.cpp
#include <iostream>
#include <omp.h>

int main() {
    int init = 10;
    int local;

    #pragma omp parallel shared(init) private(local)
    {
        #pragma omp single
        init = 10;
        #pragma omp end single
        local = init;
    }
    #pragma omp end parallel

    std::cout << "init: " << init << ", local: " << local << std::endl;

    return 0;
}

File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file270.f90
program DRB125_single_orig_no
    use omp_lib
    implicit none

    integer :: init, local

    !$omp parallel shared(init) private(local)
        !$omp single
        init = 10
        !$omp end single
        local = init
    !$omp end parallel
end program
